spec/acceptance/beaker/git/branch_checkout/branch_checkout_https.rb

```diff
@@ -33,7 +33,7 @@
     server.start
     EOF
     create_remote_file(host, '/tmp/https_daemon.rb', https_daemon)
-    #on(host, "#{ruby} /tmp/https_daemon.rb")
+    # on(host, "#{ruby} /tmp/https_daemon.rb")
```

Comment:
Seems strange to even have this? It was there before so you can keep it in if you want

Reply:
removed